test/functional/job_configs.py

```diff
@@ -58,6 +67,9 @@ def __init__(self, config, expected_to_fail, expected_num_subjobs, expected_num_
         - TOKEN: seq 0 4 | xargs -I {} echo "This is atom {}"
 
 """,
+        'nt': """
+""",
```

Comment:
This PR is more about the 'how' to have platform specific commands in functional test configs than 'what' exactly those commands would be. I am rewriting those shell commands in batch, but it's gonna take a little while. This is also good for keep each PR small. What you think?